### DISCUSSION TITLE
How to stop pausing on "First chance exceptions" / C++ EH exceptions?

### DISCUSSION THREAD

**Discussion by jamesWalker55** [NationalSecurityAgency/ghidra#7167] [Debugger]
I'm trying to debug a music program called Reaper. I've successfully attached a debugger, but whenever I click on the Reaper window, the debugger immediately pauses and shows this message:

```
(86cc.be2c): C++ EH exception - code e06d7363 (first chance)
First chance exceptions are reported before any exception handling.
This exception may be expected and handled.
```

https://github.com/user-attachments/assets/de8fa5e2-1f81-4830-a42b-403dfd7b049c

As a result, I can't actually test the program since the debugger immediately pauses execution when I do anything on the Reaper window.

Is there a way to stop Ghidra from pausing when it encounters a C++ EH exception?

**Comment by d-millar** [comment]
We don't currently have a way to do this from the GUI for the python-based debuggers.  As in windbg, the "sx" commands control events/exceptions.  If I remember correctly, "sxi eh" will set the C++ EH exception handler to "ignore".
